default.nix

```diff
@@ -1,16 +1,85 @@
-with (import (fetchTarball https://github.com/NixOS/nixpkgs/archive/fb235c98d839ae37a639695ad088d19ef8382608.tar.gz) {});
-# NOTE: when bumping nixpkgs, also update nixpkgs-src.json and .travis.yml
+{ cluster ? "mainnet" }:
+let
+  pkgs = import (import ./fetchNixpkgs.nix (builtins.fromJSON (builtins.readFile ./nixpkgs-src.json))) { config = {}; overlays = []; };
+  packages = self: {
+    inherit pkgs;
+    master_config = {
+      daedalus_darwin_url = "https://update-cardano-mainnet.iohk.io/Daedalus-installer-1.1.0.408.pkg";
+      cardano_rev = "eef095";
+      daedalus_hash = "1b0iy6c5nlsa2n2ylcc2kfm8ykkmdp6ncnx4lwwvc6f0662cf175";
+      cardano_hash = "151qjqswrcscr2afsc6am4figw09hyxr80nd3dv3c35dvp2xx4rp";
+    };
+    inherit cluster;
+    cardanoSrc = pkgs.fetchFromGitHub {
+      owner = "input-output-hk";
+      repo = "cardano-sl";
+      rev = self.master_config.cardano_rev;
+      sha256 = self.master_config.cardano_hash;
+    };
+    cardanoPkgs = import self.cardanoSrc {
+      gitrev = self.cardanoSrc.rev;
+      config = {
+        packageOverrides = pkgs: {
+          rocksdb = pkgs.rocksdb.overrideDerivation (drv: {
+            outputs = [ "dev" "out" "static" ];
+            postInstall = ''
+              ${drv.postInstall}
+              mkdir -pv $static/lib/
+              mv -vi $out/lib/librocksdb.a $static/lib/
+            '';
+          });
+        };
+      };
+    };
+    daedalus = self.callPackage ./linux.nix {};
+    rawapp = self.callPackage ./rawapp.nix {};
+    nix-bundle = import (pkgs.fetchFromGitHub {
+      owner = "input-output-hk";
+      repo = "nix-bundle";
+      rev = "630e89d1d16083";
+      sha256 = "1s9vzlsfxd2ym8jzv2p64j6jlwr9cmir45mb12yzzjr4dc91xk8x";
+    }) { nixpkgs = pkgs; };
+    desktopItem = pkgs.makeDesktopItem {
+      name = "Daedalus";
+      exec = "INSERT_PATH_HERE";
+      desktopName = "Daedalus";
+      genericName = "Crypto-Currency Wallet";
+      categories = "Application;Network;";
+      icon = "INSERT_ICON_PATH_HERE";
+    };
+    iconPath = ./installers/icons/1024x1024.png;
+    namespaceHelper = pkgs.writeScriptBin "namespaceHelper" ''
+      #!/usr/bin/env bash
 
-stdenv.mkDerivation {
-  name = "daedalus";
+      set -ex
+      cd ~/daedalus-install/
```

Comment:
This could be a top-level configurable field and default to something like `~/.daedalus` - what do you think?